pwalvarado/qeue_example.rb

```diff
@@ -0,0 +1,22 @@
+require 'thread'
+
+queue = Queue.new
+
+producer = Thread.new do
+  10.times do |i|
+    sleep 1 # simulate expense
+    queue << i
+    puts "#{i} produced"
+  end
+end
+
+consumer = Thread.new do
+  sleep 3
+  5.times do |i|
```

Comment:
Unused block argument - `i`. You can omit the argument if you don't care about it.